### PR TITLE
Improve execution time 1

### PR DESCRIPTION
**Pull request by blueberry1312**
## In this PR, We did the following changes:
- Added 8.000.000+ data to visits table.
- Added 5.000.000+ data to owners table.
**This data were added to lenghten the execution time:**
pair1
pair2
pair3

-------------------

-Improved the execution time for the three queries by denormalizing vets and animals tables:

pair1sol
image
image
image

-----------------------

**Performance with the changes requested by the reviewer:**
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -68,4 +68,12 @@ CREATE TABLE visits (
...
 	date_of_visit DATE NOT NULL
-);
\ No newline at end of file
+);
+
+--------------------------------------------------------
+
+ALTER TABLE animals ADD COLUMN visits_counter INT;
+
+CREATE INDEX vet_index ON visits(vet_id);
```
Comment (youmari) [inline schema.sql:77]: - [x] it would be better to create the index for `animal_id ` also in order to increase the performance

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @gdumani , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 2]
```diff
@@ -68,4 +68,16 @@ CREATE TABLE visits (
 	vet_id INT REFERENCES vets(id),
 	animal_id INT REFERENCES animals(id),
 	date_of_visit DATE NOT NULL
-);
\ No newline at end of file
+);
+
```
Comment (youmari) [inline schema.sql:72]: - [x] [OPTIONAL] There is a typo in the 15 line that I missed previously 
Screenshot 2023-05-09 at 18 55 48

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @blueberry1312 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

- [ OPTIONAL] the email column doesn't exist in the owners table please make sure to add it 
### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 2 above)
Comment (blueberry1312) [reply schema.sql:72]: thanks i miss that

**Pull request merged**